src/main/java/kenigsberg/wordledictionary/WordleController.java

```diff
@@ -0,0 +1,102 @@
+package kenigsberg.wordledictionary;
+
+import javax.swing.*;
+import java.awt.*;
+
+public class WordleController {
+    private final WordleGame wordleGame;
+    private final WordleDictionary wordleDictionary;
+
+    private final JLabel[][] labels;
+    private final JButton[] keyboard;
+    private final JButton enter;
+    private final JButton backspace;
+    private int guessNum;
+    private int guessLetter;
+    private boolean complete;
+    private StringBuilder theGuess;
+    private String correct;
+
+
+    public WordleController(WordleGame wordleGame, WordleDictionary wordleDictionary,
+                            JLabel[][] labels, JButton[] keyboard, JButton enter, JButton backspace) {
+        this.wordleGame = wordleGame;
+        this.wordleDictionary = wordleDictionary;
+        this.labels = labels;
+        this.keyboard = keyboard;
+        this.enter = enter;
+        this.backspace = backspace;
+        this.theGuess = new StringBuilder();
+        this.guessNum = 0;
+        this.guessLetter = 0;
+    }
+
+    /**
+     * Called when you type the letter in or press letter on keyboard AKA actionListener button
+     *
+     * @param letter - the letter to be added to the fields grid
+     */
+    public void addLetter(String letter) {
+        this.correct = wordleGame.getCorrectWord();
```

Comment:
Why?

Reply:
So that I can test for putting in the correct word - can only access the correct one using the getCorrectWord() method. - Will remove later